guides-and-policies/policy-source-control.md

```diff
@@ -43,9 +44,12 @@ At Arcadia Science, repositories should follow basic guidelines.
 - **Getting Code into Arcadia Science Repositories:** Code moves from feature branches to the main branch through a process of code review. Code review is handled through pull requests. The process is described briefly below. Feel free to ask for guidance if you are uncomfortable with the process. We will revoke write access for failing to adhere to these rules.
 - **Why do code reviews?** It increases the code quality two-fold: 1) because you know your work will be reviewed, you pay more attention and 2) because someone else (or multiple people) carefully look at a piece of code. It increases accountability and visibility. Effectively, you’re creating a paper-trail of changes and decisions around those. A potential complaint could be that it’s a time sink and to be honest it is. But the benefits outweigh that cost.
 - **What is a pull request?** Pull requests are the heart of collaboration on GitHub. When you open a pull request, you're proposing your changes and requesting that someone review and pull in your contribution and merge them into their branch. Pull requests show diffs, or differences, of the content from both branches. The changes, additions, and subtractions are shown in different colors. Want to see what it looks like? [Here’s an example](https://github.com/Arcadia-Science/2022-mtx-not-in-mgx-pairs/pull/3).
-- **Composition of Pull Requests:** Each pull request may contain one or more code changes. In keeping with good source control practice, each change or commit should contain all changes necessary for a particular fix or update. In addition, each pull request should relate to no more than one functional area in the code base you are updating. Keeping the pull request focused to one area makes it easier for your reviewers to provide thoughtful feedback.
+- **Composition of Pull Requests:** Each pull request may contain one or more code changes as long as they focus on a specific task. In keeping with good source control practice, each change or commit should contain all changes necessary for a particular fix or update. In addition, each pull request should relate to no more than one functional area in the code base you are updating. Keeping the pull request focused to one area will most likely keep them short. Short pull requests make it easier for your reviewers to provide thoughtful and quick feedback. Quick feedback means you can quickly address concerns and get your pull request merged. In general, shorter and more focused pull requests are better than longer ones.
```

Comment:
I think this is fine, but this reads more like a software engineering set of recommendations than a data science one. I think it can be hard to scope a PR for a data analysis project, and to a certain extent I think it's ok that the size of these things will vary.

One rec I think could be good to add is that all of the code in a PR should have been executed before. This is sort of obvious for a software repo which has CI tests, but isn't necessarily the case for a data analysis repo.

Reply:
sorry that wasn't super actionable feedback. I don't necessarily think this needs to change now, but we might need to keep an eye on this and pay attention to how people best interact with PRs from a data sci perspective and update eventually.